Themes/default/languages/ManageSettings.english.php

```diff
@@ -202,6 +202,7 @@
 $txt['recaptcha_secret_key_desc'] = 'This is for communication between your site and Google. Be sure to keep it a secret.';
 $txt['recaptcha_no_key_question'] = 'Don\'t have a key for reCAPTCHA?';
 $txt['recaptcha_get_key'] = 'Get your reCAPTCHA key here.';
+$txt['languages_recaptcha'] = '<a href="https://developers.google.com/recaptcha/docs/language" target="_blank" class="bbc_link">ReCAPTCHA language</a>';
```

Comment:
Can't this be formatted in other way?

Reply:
Oh, fine, if you're going to be all picky about it, I'll make a $helptxt entry for it. 😜